docs/intro-to-coalton.md

```diff
@@ -548,6 +548,16 @@ All of these cases are sufficiently common that we provide a few shorthands:
 
 - `floor/`, `ceiling/`, and `round/` for integer-to-integer division, and
 
+Fractions can be converted to other dividable types using `fromfrac` (Note: This may result in precision loss):
+
+```
+COALTON-LIBRARY/MATH/REAL> (coalton (the Double-Float (fromfrac (/ 1 2))))
```

Comment:
you can just write 1/2 and 999/1000 and the like, right?

Reply:
Yup! changed them and checked them